_data/events.yml

```diff
@@ -1,14 +1,9 @@
-- date: 2020-04-08
-  name: "Virtual Meetup- AGA v2.0 (*Ask Gian Anything*)"
-  info: "Hosted by the Bay Area Apache Druid Meetup Group"
-  link: https://bit.ly/3dMtdtl
-  
-- date: 2020-04-09
-  name: "Virtual Meetup- Apache Druid 101: Analytics for Real-time Intelligence"
-  info: "Hosted by the Chicago Apache Druid Meetup Group"
-  link: https://bit.ly/39GE453
-  
-- date: 2020-04-13
-  name: "Virtual Druid Summit"
-  info: "The comfort of your own laptop"
+- date: 2020-05-27
+  name: "Virtual Apache Druid Meetup: AIADA (Ask Itai and David Anything)"
+  info: "Hosted by Big Things Meetup Group"
+  link: https://www.meetup.com/Big-things-are-happening-here/events/270646030/
+
+- date: 2020-11-02
+  name: "Druid Summit"
+  info: "San Francisco Airport Marriott Waterfront, 1800 Old Bayshore Highway, Burlingame, CA"
```

Comment:
suggest to add 
`link: https://druidsummit.org/`

Reply:
@surekhasaharan - it's actually already there, look at the line below :)

Reply:
oh i overlooked, it's good to go then, thanks for the PR.